ui/components/component-library/input/input.types.ts

```diff
@@ -0,0 +1,6 @@
+export enum InputType {
+  Text = 'text',
+  Number = 'number',
+  Password = 'password',
+  Search = 'search',
+}
```

Comment:
Need a line break here. As well as the `InputProps`
```suggestion
}
```